File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ChartData.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ChartData 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public string Identifier { get; set; }
    public string IdAttribute { get; set; }
    public string Label { get; set; }
    public long MaxValue { get; set; }
    public long TotalCount { get; set; }
    public List<Object> Items { get; set; }
    public List<Object> AggregatedItems { get; set; }
    public string Caption { get; set; }
    public string ShowAnchor { get; set; }
    public string AnchorAlpha { get; set; }
    public string GetxAxisName { get; set; }
    public string GetyAxisMinValue { get; set; }
    public string GetyAxisName { get; set; }
    public string DecimalPrecision { get; set; }
    public string FormatNumberScale { get; set; }
    public string NumberPrefix { get; set; }
    public string ShowNames { get; set; }
    public string ShowValues { get; set; }
    public string ShowAlternateHGridColor { get; set; }
    public string AlternateHGridColor { get; set; }
    public string DivLineColor { get; set; }
    public string DivLineAlpha { get; set; }
    public string AlternateHGridAlpha { get; set; }
    public string RotateNames { get; set; }
    public string Returning { get; set; }
}

File: server-scala-play-server/app/model/ChartData.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ChartData.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ChartData(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  identifier: Option[String],
  idAttribute: Option[String],
  label: Option[String],
  maxValue: Option[Long],
  totalCount: Option[Long],
  items: Option[List[JsObject]],
  aggregatedItems: Option[List[JsObject]],
  caption: Option[String],
  showAnchor: Option[String],
  anchorAlpha: Option[String],
  getxAxisName: Option[String],
  getyAxisMinValue: Option[String],
  getyAxisName: Option[String],
  decimalPrecision: Option[String],
  formatNumberScale: Option[String],
  numberPrefix: Option[String],
  showNames: Option[String],
  showValues: Option[String],
  showAlternateHGridColor: Option[String],
  alternateHGridColor: Option[String],
  divLineColor: Option[String],
  divLineAlpha: Option[String],
  alternateHGridAlpha: Option[String],
  rotateNames: Option[String],
  returning: Option[String]
)

object ChartData {
  implicit lazy val chartDataJsonFormat: Format[ChartData] = Json.format[ChartData]
}

